src/runtime_src/core/pcie/driver/linux/xocl/xocl_fdt.c

```diff
@@ -196,7 +196,7 @@ static void devinfo_cb_plp_gate(void *dev_hdl, void *subdevs, int num)
 {
 	struct xocl_subdev *subdev = subdevs;
 
-	subdev->info.level = XOCL_SUBDEV_LEVEL_BLD;
+	subdev->info.level = XOCL_SUBDEV_LEVEL_PRP;
```

Comment:
it should be BLP. plp_gate is implemented in BLP and it isolates BLP.

Reply:
Thanks, adjust it back. Actually the clock.c should have use BLD instead of PRP.